infra/modules/gcp-bulk/variables.tf

```diff
@@ -29,3 +29,8 @@ variable "source_kind" {
   type        = string
   description = "Kind of the content to process"
 }
+
+variable "psoxy_basedir" {
+  type        = string
+  description = "the path where your psoxy repo resides"
```

Comment:
add a default value, but imho should be split out (path to jar, path to config, etc)